README.md

```diff
@@ -1,2 +1,50 @@
 
-NoSql Assignment
+NoSQL Assignment 
+Users 
+ 
+{ 
+_id: “username”, 
+name: “Full name here”, 
+email : “​ email_here@example.com​ ”, 
+password: “store pass hash here”, 
+bio : “user self info”, 
+teams : [ 
+{ 
+name: “team name”, 
+role: “admin, super or editor” 
+}, ... 
+]  
+ 
+} 
+ 
+Teams  
+Assumption 1 : only one admin and editor per team, otherwise would use array 
+Assumption 2 : team names are unique 
+{ 
```

Comment:
Level of complexity to find out if a particular user is a particular role for a team is very high. 
Reduce this complexity and stick to the original problem statement.